components/event-bus/cmd/event-bus-publish-knative/handlers/publish_test.go

```diff
@@ -2,6 +2,8 @@ package handlers
 
```

Comment:
I suggest to move this file to `components/event-bus/internal/knative/util/util_test.go`.

Reply:
Done